build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+# Build x86 images
+docker build --build-arg IMAGEARCH= \
+                   --build-arg QEMUARCH="x86_64" \
+                   --file Dockerfile --tag sgerrand/alpine-abuilder:amd64 . --pull
```

Comment:
🙋 The images for this project are currently tagged by Alpine version. I'd prefer to keep that prefix in front of any architecture related tag, such as `amd64`.